src/web/components/Core/TermsAndConditions.scss

```diff
@@ -0,0 +1,34 @@
+.terms-and-conditions {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .text-button {
+    margin-top: 10px;
+  }
+
+  .terms-container {
+    border: 1px solid var(--theme-border);
+    border-radius: 5px;
+    padding: 16px;
+    height: 468px;
+    overflow: scroll;
+    margin-bottom: 22px;
+    max-width: 910px;
```

Comment:
This needs to be wider than the Figma design to look right - we might end up narrowing it later but we'll see.